From 476155e39ec420f23f087a4257d7356071351c2a Mon Sep 17 00:00:00 2001
From: Michal Klocek <michal.klocek@qt.io>
Date: Thu, 3 Jul 2025 10:34:21 +0200
Subject: [PATCH] Fix compilation in ci with lastest security patches

Add missing header.

Deals with:
"blink/renderer/core/workers/worker_thread.cc:301:28:
error: 'kWorkerThreadSequentialShutdown' is not a member of 'blink::features'"

Change-Id: Ic85ea4329c6c82c59044b2d8e8033913823148aa
Upstream-Status: Submitted
---
 .../third_party/blink/renderer/core/workers/worker_thread.cc     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/third_party/blink/renderer/core/workers/worker_thread.cc b/chromium/third_party/blink/renderer/core/workers/worker_thread.cc
index cb50e1ceca3..f53d4c28a58 100644
--- a/chromium/third_party/blink/renderer/core/workers/worker_thread.cc
+++ b/chromium/third_party/blink/renderer/core/workers/worker_thread.cc
@@ -35,6 +35,7 @@
 #include "base/synchronization/waitable_event.h"
 #include "base/task/single_thread_task_runner.h"
 #include "base/threading/thread_restrictions.h"
+#include "third_party/blink/public/common/features.h"
 #include "third_party/blink/public/common/loader/worker_main_script_load_parameters.h"
 #include "third_party/blink/public/mojom/frame/lifecycle.mojom-shared.h"
 #include "third_party/blink/public/platform/platform.h"
